docs/ha-scalable-wordpress.md

```diff
@@ -510,7 +525,333 @@ Follow this guide to set up a high availability (HA) and scalable WordPress depl
 
 - Enjoy!
 
+
+## Backup and Restore
```

Comment:
Back up

Reply:
Backup is a "nound" # "Back up" is a verb phrase.

Reply:
they're both nouns here